core/templates/domain/exploration/ExplorationMetadataObjectFactory.ts

```diff
@@ -20,26 +20,29 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
-export class ExplorationMetadata {
-  metadataList;
-
-  constructor(metadataList) {
-    this.metadataList = angular.copy(metadataList);
-  }
+export interface ExplorationMetaDataBackendDict {
+  id: string;
+  objective: string;
+  title: string;
+}
 
-  getMetadataList() {
```

Comment:
Just to confirm, this used to store an array of objects and is now changed to a single object, so all its calling functions have been edited correctly, right?

Reply:
Yup, it was used in the collection-node-creator page. I've refactored it accordingly.